.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: test
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: "45 3 * * 6" # Runs at 03:45, only on Saturday
```

Comment:
This scheduled run is to fulfill the requirements for Awesome Crystal guideline. 
Please see https://github.com/veelenga/awesome-crystal/blob/master/.github/CONTRIBUTING.md